LenderToolkit/Skills-Test-React
# Build fix

**Ticket:** [URL of the ticket]

## Description:
skills-test-react fix

## Solution:
This code adds the form with the /todo toggle

## Customer Impact:
[If this change is behind a feature flag/toggle or not a new feature, you may opt to skip this section.]
[Short paragraph explaining how the customer will be impacted by this change. Try answering questions like: What will they see or experience? Will anything they experience cause concern, anxiety, or mistrust in the platform? What can we do to eliminate those concerns?]

## QA Testing Guidelines:
Enter the todo at /todo and delete the todo to test functionality.

Checklist:

Minimally check off the following to indicate you've read & considered them. Add notes as appropriate.

- [ ] [Specs] Do you have unit/feature tests? Is code coverage sufficient?

- [ ] [Feature flags] Are the changes behind a feature flag? (If applicable)

- [ ] [Docs] Is there any documentation needed? (Tooltips, Help Article links, etc.)

- [ ] [Security] Did you add or edit a controller endpoint?


- [ ] [Monitoring] Can we detect and troubleshoot performance issues or bugs?

  - [ ] Are there timings for slow operations? Is app state logged for failures?

  - [ ] Do we need to add new Datadog monitors, graphs, or entire dashboards?
